code/game/objects/configuration.dm

```diff
@@ -0,0 +1,41 @@
+/obj/configuration

```

Comment:
Errr, no, use the map config json for this kind of stuff

Reply:
I tried asking you multiple times about how to configure maps if they had no attached metadata... :(

So this was the best method I had available to me. The objects themselves are placed near the shuttles that they'll be loading in.